### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by lipki** [SummerFields/SummerFields#718] [General]
![General!](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fs-media-cache-ak0.pinimg.com%2Foriginals%2Fe5%2Fdc%2Fec%2Fe5dcec203cad6477a7f4cf3c82691f83.jpg&f=1&nofb=1)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pfccalico** [comment]
PFC Calico, just returning to see if you still wanted help with MC Bedrock, I have still been working on little bits here and there with it between my university stuff. I have a Zombie that blinks and moves it's mouth and carrots and potatoes that randomly flower at max growth level. I see you already have randomised textures! I can also show some of the animated textures I have done as well for Bedrock if you like.

**Reply by lipki** [reply]
Of course we want help.
I'm still not working on the Bedrock, but @chengzzi89  will surely not refuse your help.
And I'm always interested in random textures.

**Reply by pfccalico** [reply]
the random textures are just the ones from the main java pack that have been made usable for bedrock, but i think chengzzi has done that one already but i can send across what i have done for the entity to get them to animate

**Comment by lipki** [comment]
It's sort of reddit here actually !?

**Reply by lipki** [reply]
j'ai jamais rien compris a reddit.

**Comment by CuddleWoozle** [comment]
I'm not really sure how to put this...

I know I haven't helped out on this in a very _very_ long time, and it's not likely that I'm going to start helping again anytime soon.

Do you need me to say something official like "CuddleWoozle isn't working on SummerFields anymore and gives control of it's fate to the good people who have spent years working on it"?  Or is having released it under the share-alike license good enough for that?

I can keep it up on PMC if you want me to, but I'll change the links/files to point to the website and this Github repository.  (I already added the links, I would basically be removing the download there if possible.  The pack is basically too large to properly upload to the site the last time I tried.)

**Reply by lipki** [reply]
CuddleWoozle !!
:heart:
I tried to join you anywhere on the internet, but I didn't want to stalk you.
Hope you are doing well, and that everything is going well in your life right now.

**Reply by lipki** [reply]
So yes, as LithiumSound does it for you ...
I would like an official announcement from you which transfers full powers to **me**.
 
I know this is absurd, but it gave me problems not to be official. It might do me a favor in the future.

**Reply by lipki** [reply]
I would like access to PMC. (password)

And also if you have any historical textures on your computer, I know you worked on horses at one point.
And I imagine you must have some unused variants or source files.
I am very fond of it.

**Reply by CuddleWoozle** [reply]
I'm doing better.  I got stressed/major anxiety issues and had to pretty much abandon social media.  I got burnt out on art entirely and only earlier this year started doing artwork again and am doing my own texture pack just for my own tastes.  (It's not released.  I'm posting 1 screencap a day from it on my art gallery site until it's entirely finished.  The feedback helps motivate me!)

Alas, I cannot give you my PMC account.  It's on my personal account and I don't have the password for the old account on there.  The other account is LithiumSounds and I don't think he ever gave me the password for his account.  If you make a new account, I can remove the submission from mine/post you permission there, too.

The last textures I did for SummerFields were several laptops ago and I don't have anything left that wasn't already published online.  (The laptop hard drive died and took what I had with it.  :( )

Here's your official permission:

--Let it be known that I, CuddleWoozle, give Lipki permission to continue working on SummerFields so long as the pack remains freely available to everyone.--

**Reply by lipki** [reply]
The license is my only lord, and I would respect it over my life. 🖖

😄

I saw your first textures, I see you to start from scratch, that gives you a lot of freedom.
And much less stress :D

Come see us from time to time anyway.
I'm starting to do some community management on Twitter / Instagram / Discord.

I made a PMC account.
https://www.planetminecraft.com/member/summerfields_team/
I also made a first submission, to see if it passes.
We'll see.

**Reply by CuddleWoozle** [reply]
I'll still be poking in from time-to-time, I'm sure.  :)

And if they post your submission, I will comment to verify that you do have permission from me so there is no drama about it.  XD

**Reply by lipki** [reply]
It doesn't worry me.

**Reply by chengzzi89** [reply]
😃
